Mobl_ios_Callisto-master/MealTrackingPilotTests/Mocks/MockLoginClient.swift

```diff
@@ -26,7 +25,7 @@ class MockLoginClient: LoginCredentialsLoginClient {
         functionCalls.append("logout")
     }
 
-    func refreshLogin(completion: ((Result<AccessCredentials>) -> Void)?) {
+    func refreshLogin(completion: ((Result<AccessCredentials,Error>) -> Void)?) {
```

Comment:
Replace value ((Result<AccessCredentials,Error>) -> Void) to accessCredentialsCompletion

Reply:
Done